### ISSUE TITLE
Docs: Add "Getting Started" guide to README

### ISSUE THREAD

**Issue by BOLT04** [codemortals/game-platform#34]
Hi 👋, 

I was trying to set up this project for local development after I cloned the repo, but I found some difficulty in doing so. I think mostly it was just that I don't know Firebase that well and I also got this error after running `npm start`:
![imagem](https://user-images.githubusercontent.com/18630253/83355059-cec7c380-a354-11ea-8b9f-9c93a59b2656.png)

Then I realized the instructions to create that file wherein `environment.ts`, so I followed them and got the Angular app running locally 😃 .

Perhaps we could add some documentation on the README explaining the steps and the commands the person needs to run, and some docs on the project structure. This could help people that are not familiar with Angular and can't find the instructions that are on the code. For example the following:

### Local Development for `/functions`
1. Install [Firebase CLI](https://firebase.google.com/docs/cli) (**Required**)
...
### Local Development for `/hosting`
1. Clone the project or your fork (if you plan to make changes use your fork)
2. Install dependencies by running the command `npm install`
```bash
git clone https://github.com/codemortals/game-platform.git
cd game-platform/hosting
npm install
```
3. Create a [Firebase](https://firebase.google.com/) account and a new project
   3.1. Goto **Project Overview** and Add a "Web App", call this as you please for example; CodeMortals Development. If you need extra help, have a look at [these docs](https://firebase.google.com/docs/web/setup)
   3.2 Create a file called `environment.local.ts` in `/hosting/src/environments` and copy your app configuration in there

What do you think of this addition @andrew-cunliffe and @eddiejaoude ?

**Comment by eddiejaoude** [comment]
OMG I didnt event realise the `README.md` didnt have a QuickStart part in it. That is my fault, as I think I said I would do it and forgot 🤦‍♂️ .

Thank you @BOLT04 👍 . I will do one now, and we can improve on it incrementally 🤓

**Comment by eddiejaoude** [comment]
I made a start on it, let me know what you think. Thanks @BOLT04 👍

**Comment by BOLT04** [comment]
Yeah, looks good @eddiejaoude. Maybe we could add a few links to Firebase docs and on `4b` of the Firebase steps or before that, we could say that the Firebase CLI is required. If they don't have it installed and try to run `npm start` after `npm install`, since the `serve` script uses that tool it will produce an error.

**Comment by eddiejaoude** [comment]
I added a line for this, thanks 👍  - I am sure we can improve it more with time 🤓
